pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["src"]
+
+[project]
+name = "veritaserum"
+version = "0.0.1"
+
+[options]
+python_requires = ">=3.6"
```

Comment:
i would use a more recent version so we can use recent language features (at least 3.10)

Reply:
Done